app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
```

Comment:
We should get more explicit about their pain point. They don't necessarily care if it's "welcoming", they just want contributions.

_"Take one day this week to make your project easy to contribute to"_